stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -115,6 +115,10 @@ namespace stop_and_wait_plugin
             }
         }
 
+        if(current_downtrack < maneuver_plan[0].stop_and_wait.start_dist){
```

Comment:
Is there any possibility that meneuver_plan will not have at least one element? If so this can fail silently with a segfault. If we can guarentee this will not happen this is fine. Otherwise we should throw an exception here.

Reply:
Since the plan delegator will only request stop&wait for a plan, if there is an associated maneuver, there shouldn't be a case where this is zero

Reply:
Ok sounds good.